app/connector/support/maven-plugin/src/main/java/io/syndesis/connector/support/maven/GenerateConnectorInspectionsMojo.java

```diff
@@ -102,6 +117,78 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         }
     }
 
+    private void initJsonSchema() throws MojoExecutionException {
+        if (jsonSchema != null) {
+            return; // already initialised
+        }
+
+        InputStream schemaStream = null;
+        try {
+            schemaStream = getClass().getResourceAsStream("/" + CONNECTOR_SCHEMA_FILE);
+            if (schemaStream == null) {
+                throw new IOException("Schema Initialisation Error: file not available");
```

Comment:
Will this ever happen? Won't the tests fail if the `connector-schema.json` is not present.

Reply:
It did up until it the schema file was put in the correct location. Just being thorough out of habit, I guess.